src/Api/Controllers/OrganizationSponsorshipsController.cs

```diff
@@ -67,11 +65,18 @@ await _organizationSponsorshipRepository
                 (await CurrentUser).Email);
         }
 
+        [HttpPost("validate-token")]
+        [SelfHosted(NotSelfHostedOnly = true)]
+        public async Task<bool> PreValidateSponsorshipToken([FromQuery] string sponsorshipToken)
```

Comment:
We don't usually use query params in combination with post. Any particular reason?

Reply:
I wanted to match the `redeem` endpoint. Now as to why that one has a query parameter, the best I can remember is I wanted a symmetry between the link we give in the email offer and the endpoint, but that feels quite flimsy now...

Reply:
Fair enough.